src/Triggers.sol

```diff
@@ -1,6 +1,44 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.13;
 
+enum BlockRangeKind {
+    BOUNDED,                // (custom start, custom end)
+    FROM_BLOCK,             // (custom start, infinity)
+    FULL_SYNC               // (earliest available, infinity)
```

Comment:
Let's take inspiration from how Rust ranges are named: https://doc.rust-lang.org/std/ops/index.html#structs
Maybe Range/Bounded, Full, From, FromInclusive, To, ToInclusive (we can skip the inclusive ones for now)?

Reply:
Fixed in ad0a892065ec09994e5f44d2ca45d9bdae792731